.github/workflows/ci.yml

```diff
@@ -0,0 +1,28 @@
+name: CI
+on:
+  push:
+    branches:
+      - master
+      - "[0-9].[0-9]+.x"
+  pull_request: {}
+jobs:
+  checks:
+    name: checks
+    runs-on: ubuntu-latest
+    steps:
+    # the skip-duplicate-actions step actually applies to the whole workflow
+    - uses: fkirc/skip-duplicate-actions@master
+      name: cancel previous runs
+      with:
+        github_token: ${{ github.token }}
+    - uses: actions/checkout@v2
+    - uses: actions/setup-java@v1
+      with:
+        java-version: 8
+    - uses: eskatos/gradle-command-action@v1
+      name: gradle
+      with:
+        arguments: check javadoc --no-daemon
```

Comment:
why `--no-daemon`, btw? :)

Reply:
because the machine will die at end of run anyway, so there is no point in spinning up a daemon and communicating with it (there won't be any reuse)

Reply:
Have you measured this "optimization"?

Reply:
I think disabling the daemon on the CI build is appropriate because of this

```
... you can disable it to use a fresh runtime for each build since the runtime is completely isolated from any previous builds.
```

Reply:
> Have you measured this "optimization"?

No, but this is a pretty common setting in CI, so I assume it makes sense.

Reply:
I have never seen a recommendation by the Gradle team to disable it on CI, nor they do it themselves:
https://github.com/gradle/gradle/blob/91d3ab908212b6e4fd47ad2f012fd6d3cf881bc8/.travis.yml#L15

Also, if the memory settings are customized (build, test, other tasks), it would start a daemon-ish process anyway. 

I would recommend to stick to the defaults unless a proof is provided that it makes things faster/better.